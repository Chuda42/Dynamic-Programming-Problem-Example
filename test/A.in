leer_datos
10 .
leer_rendimiento
20 .
max_datos_procesados
planificacion_optima
leer_datos
10 .
leer_rendimiento
5 .
max_datos_procesados
planificacion_optima
Fin
