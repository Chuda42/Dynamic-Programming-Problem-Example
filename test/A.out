1>2>3>Max Datos Procesados: 10
4>Cantidad Datos Planificacion: 10
5>6>7>Max Datos Procesados: 5
8>Cantidad Datos Planificacion: 5
9>Fin.
